<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSViewWidget</class>
 <widget class="QFrame" name="VSViewWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization Window</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="3" column="2">
    <widget class="VSVisualizationWidget" name="visualizationWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="btnWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="btnLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="transformTextLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="splitHorizontalBtn">
        <property name="toolTip">
         <string extracomment="Split Horizontally">Split Horizontally</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../Resources/VtkSIMPL.qrc">
          <normaloff>:/icons/pqSplitViewH12.png</normaloff>:/icons/pqSplitViewH12.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="splitVerticalBtn">
        <property name="toolTip">
         <string extracomment="Split Vertically">Split Vertically</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../Resources/VtkSIMPL.qrc">
          <normaloff>:/icons/pqSplitViewV12.png</normaloff>:/icons/pqSplitViewV12.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="toolTip">
         <string extracomment="Close View">Close</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../icons/images/Icons.qrc">
          <normaloff>:/delete_plain.png</normaloff>:/delete_plain.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VSVisualizationWidget</class>
   <extends>QWidget</extends>
   <header>VSVisualizationWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Resources/VtkSIMPL.qrc"/>
  <include location="../../../icons/images/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
